bcipy/acquisition/devices.py

```diff
@@ -117,6 +117,18 @@ def analysis_channels(self) -> List[str]:
             filter(lambda channel: channel not in self.excluded_from_analysis,
                    self.channels))
 
+    def to_dict(self) -> dict:
+        """Converts the DeviceSpec to a dict."""
+        return {
+            'name': self.name,
+            'content_type': self.content_type,
+            'channels': [ch._asdict() for ch in self.channel_specs],
+            'connection_methods': [cm.name for cm in self.connection_methods],
```

Comment:
I'm going to be deprecating `connection_methods` soon, so you could probably leave that property off. Otherwise I can remove it when I do the rest.